types/api/custom_profile_attributes.d.ts

```diff
@@ -22,6 +22,7 @@ type CustomProfileField = {
     /** any extra properties of the field **/
     attrs?: {
         sort_order?: number;
+        saml?: string;
```

Comment:
Q: what is the actual value of saml?

Reply:
a unique identifier, it is a string returned from the API in an optional parameter.